bouncer/app.py

```diff
@@ -20,10 +21,18 @@ def settings():
     else:
         debug = False
 
+    extension_ids = os.environ.get(
+        "CHROME_EXTENSION_ID", "bjfhmglciegochdpefhhlphglcehbmek"
+    )
+    if extension_ids.strip().startswith("{"):
+        extension_ids = json.loads(extension_ids)
+        if not extension_ids.get("default"):
+            raise Exception('CHROME_EXTENSION_ID map must have a "default" key')
+    else:
+        extension_ids = {"default": extension_ids}
```

Comment:
Alternatively you could do:

```python
try:
    extension_ids = json.loads(os.environ.get("CHROME_EXTENSION_ID", "..."))
except ValueError:  # (Or whatever exception json.loads raises)
    extension_ids = {"default": extension_ids}

if "default" not in extension_ids:
    raise RuntimeError("...")
```

Alternatively you could add a new envvar named `CHROME_EXTENSION_IDS` (plural) and have the code simply assume the envvar is present and is a valid JSON object and crash otherwise. Add the new envvar, then deploy the code, then remove the old envvar.

Reply:
Thanks for the suggestions. Having a separate env var would have the advantage that the expected format is clear, though we'd need to define the precedence if both `CHROME_EXTENSION_ID` and `CHROME_EXTENSION_IDS` are set.